docs/source/dev-guide/deprecations.md

```diff
@@ -0,0 +1,65 @@
+[cep9]: https://github.com/conda-incubator/ceps/blob/main/cep-9.md
+
+# Deprecations
+
+Conda abides by the Deprecation Schedule defined in [CEP-9][cep9]. To help make deprecations as much of a no-brainer as possible we provide several helper decorators and functions facilitate the correct deprecation process.
+
+## Functions, Methods, and Properties
+
+The simplest use case is for deprecating any function, method, or property:
+
+```python
+from conda.deprecations import deprecated
+
+
+@deprecated("23.9", "24.3")
+def foo():
+    ...
+```
+
+As a minim we must always specify two versions:
+
+1. the future deprecation release in which the function, method, or property will be marked as deprecated, prior to that the feature will show up as pending deprecation (which we treat as a commenting period), and
+2. the subsequent deprecation release in which the function, method, or property will be removed from the code base.
+
+Additionally, you may provide an addendum to inform the user what they should do instead:
+
+```python
+from conda.deprecations import deprecated
+
+
+@deprecated("23.9", "24.3", addendum="Use `bar` instead.")
+def foo():
+    ...
+```
+
+## Keyword Arguments
+
+!!! warning
+    Deprecating or renaming a positional argument is unnecessarily complicated and so it isn't supported. It is recommended to either (1) devise a custom way of detecting usage of a deprecated positional argument (e.g., type checking) and using the `conda.deprecations.deprecated.topic` function (see [Topics](#topics)) or (2) deprecate the function/method itself and defining a new function/method without the deprecated argument.
```

Comment:
```suggestion
> :warning: **Deprecating or renaming a positional argument is unnecessarily complicated and so it isn't supported.** It is recommended to either (1) devise a custom way of detecting usage of a deprecated positional argument (e.g., type checking) and using the `conda.deprecations.deprecated.topic` function (see [Topics](#topics)) or (2) deprecate the function/method itself and defining a new function/method without the deprecated argument.
```
Even though the `!!! warning` syntax usually works on Markdown documents, it doesn't seem supported in GitHub. Please see the screenshot below for how it renders:
![Screen Shot 2023-03-13 at 4 01 57 PM](https://user-images.githubusercontent.com/28930622/224821539-c696fa54-3334-4228-be4b-0366eb619824.png)

Indenting the line and adding a warning emoji ⚠️ is the alternative syntax being suggested here.